### ISSUE TITLE
Hawtio-integration with Camel 3.0.1

### ISSUE THREAD

**Issue by caroline-harvey** [hawtio/hawtio-integration#97]
When is the schedule release plan for Camel 3.0.1 support?

**Comment by jamesnetherton** [comment]
Last time I checked, the Camel plugin already works fine with Camel 3.0.x and the upcoming 3.1.0 release.

**Comment by caroline-harvey** [comment]
Thanks for your quick reply,
After the upgrade to 3.0.1, we noticed that tracing was disabled by default so we enabled it.  The Hawtio Trace tab was not visible before adding these 2 properties and now it is but no tracing details.
camel.springboot.tracing=true
camel.springboot.backlog-tracing=true





Version used:

<springboot-version>2.2.4.RELEASE</springboot-version>
<camel-version>3.0.1</camel-version>





pom.xml dependency:


<dependency>
   <groupId>io.hawt</groupId>
   <artifactId>hawtio-springboot</artifactId>
   <version>2.8.0</version>
</dependency>
<dependency>
   <groupId>org.springframework.boot</groupId>
   <artifactId>spring-boot-starter-actuator</artifactId>
</dependency>
<dependency>
   <groupId>io.micrometer</groupId>
   <artifactId>micrometer-core</artifactId>
</dependency>
<dependency>
   <groupId>io.micrometer</groupId>
   <artifactId>micrometer-registry-prometheus</artifactId>
</dependency>



Caroline

**Comment by jamesnetherton** [comment]
That configuration works fine for me. I have a simple timer route to generate messages every 5 seconds and I see them show up in the tracing tab.

```
from("timer:foo?period=5s")
    .log("Hello world");
```

There is a bit of a problem with the UI in that the ID table column is empty and the table gets so large that it shrinks the Camel route diagram. I'll log that as a separate issue.

![traces](https://user-images.githubusercontent.com/4721408/75037819-f6cd9e00-54ac-11ea-805c-5e7bcbd08ccc.png)

**Comment by caroline-harvey** [comment]
So basically you have the same behavior as me where you can’t drill in the trace log to see details/payloads.
We used to see the details

[cid:image001.jpg@01D5E892.8352AB40]


Caroline

**Comment by caroline-harvey** [comment]
This is what we want, being able to see the details like we used to ☺
Do you know why this functionality disappeared?

Caroline
From: Harvey, Caroline CJ [NC]
Sent: February-21-20 8:40 AM
To: 'hawtio/hawtio-integration' <reply@reply.github.com>; hawtio/hawtio-integration <hawtio-integration@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [hawtio/hawtio-integration] Hawtio-integration with Camel 3.0.1 (#97)

So basically you have the same behavior as me where you can’t drill in the trace log to see details/payloads.
We used to see the details

[cid:image002.jpg@01D5E892.B4AE9FF0]


Caroline

**Comment by jamesnetherton** [comment]
I think the missing part is this configuration:

```
camel.springboot.use-breadcrumb=true
```

**Comment by caroline-harvey** [comment]
It works!  Thanks for your support!

Caroline
